### DISCUSSION TITLE
Questions about `KHR_texture_transform` interactions with `KHR_mesh_quantization`

### DISCUSSION THREAD

**Discussion by sotrh** [zeux/meshoptimizer#727] [Q&A] [answered]
I'm using `gltfPack` to optimize some of the models we are using and I got the positions and normals to render correctly, but the texture coordinates are messed up. Loading the file into blender works just fine, so it's clear that I'm not handling the texture coordinates properly.

I decided to look at the model itself and noticed that the material has the `KHR_texture_transform` section. I've listed it below:

```json
"KHR_texture_transform": {
    "offset": [
        0,
        0
    ],
    "scale": [
        16.003664,
        16.003664
    ]
}
```

That `scale` value looks really weird to me. Does this seem correct?

**Comment by zeux** [comment, marked as answer]
The scale depends on the UV range in the source model but 16 is a plausible value. Note that the accessors that gltfpack uses for texture coordinates use normalized attribute, so the i16 values from the buffer are implicitly divided by 65535 per spec.

**Reply by sotrh** [reply]
Ok good to know
